build/Makefile

```diff
@@ -101,6 +106,13 @@ ifdef DOCKER_RUN
 	ensure-build-image += ensure-build-image
 endif
 
+ifdef INCREMENTAL_BUILD
```

Comment:
I'm thinking that this should be defaulted on - and then turned off for builds in `do-release`. I'm not seeing any downsides. WDYT?

Reply:
switched default to incremental.